src/status_im2/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -25,31 +26,23 @@
 
 (defn page-title
   []
-  [rn/view {:style style/title-container}
-   [quo/text
-    {:accessibility-label :notifications-screen-title
-     :weight              :semi-bold
-     :size                :heading-1
-     :style               {:color colors/white}}
-    (i18n/label :t/intro-wizard-title6)]
-   [quo/text
-    {:accessibility-label :notifications-screen-sub-title
-     :weight              :regular
-     :size                :paragraph-1
-     :style               {:color colors/white}}
-    (i18n/label :t/enable-notifications-sub-title)]])
+  [quo/title
+   {:title                        (i18n/label :t/intro-wizard-title6)
+    :title-accessibility-label    :notifications-title
+    :subtitle                     (i18n/label :t/enable-notifications-sub-title)
+    :subtitle-accessibility-label :notifications-sub-title}])
 
 (defn enable-notification-buttons
-  []
-  [rn/view {:style style/enable-notifications-buttons}
+  [{:keys [insets]}]
+  [rn/view {:style (style/buttons insets)}
    [quo/button
     {:on-press                  (fn []
                                   (rf/dispatch [::notifications/switch true platform/ios?])
                                   (rf/dispatch [:init-root :welcome]))
      :type                      :primary
      :before                    :i/notifications
      :accessibility-label       :enable-notifications-button
-     :override-background-color (colors/custom-color :magenta 60)}
+     :override-background-color (colors/custom-color :purple 60)}
```

Comment:
@alwx button colors in these screens are determined by the profile color. I think you might have to update this line to something like this code so that the button is updated with the color selected by the user when they are creating their profile 
```clojure
:override-background-color (colors/custom-color (:color (rf/sub [:onboarding-2/profile]) 60)
```